docs/README-migration.md

```diff
@@ -28,6 +28,8 @@ CFLAGS += $(shell pkg-config sdl3 --cflags)
 LDFLAGS += $(shell pkg-config sdl3 --libs)
 ```
 
+The SDLmain and SDLtest libraries have been renamed SDL_main and SDL_test, respectively.
```

Comment:
I suppose we don't care about the version number here, and in comments?

Reply:
No, leaving the version off is fine.

Reply:
Actually, maybe we do want it, since it'll be the name of the library they link with.